index.js

```diff
@@ -201,10 +201,15 @@ module.exports = class Hyperdrive extends EventEmitter {
     return this.files.put(normalizePath(name), { executable: false, linkname: dst, blob: null, metadata })
   }
 
-  entry (name) {
-    return typeof name === 'string'
-      ? this.files.get(normalizePath(name))
-      : Promise.resolve(name)
+  async entry (name, opts) {
+    if (typeof name !== 'string') return Promise.resolve(name)
+
+    try {
+      return await this.files.get(normalizePath(name), opts)
+    } catch (err) {
+      if (err.message === 'Block not available locally') return null
```

Comment:
where is this message defined?

Reply:
Unrelated, I think we should change this one (removing `0`):
https://github.com/holepunchto/hyperbee/blob/317d565c3431fada1c8e16f71e4465ca4fd92dfd/index.js#L468

Reply:
And we should add error codes!